### PR TITLE
GIX-1985: Remove ENABLE_MY_TOKENS simple

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Remove `ENABLE_MY_TOKENS` flag.

In this PR, remove the usage of the flag when it's limited to that specific file and fixing the spec file means only removing test cases.

# Changes

* Remove usage in IcrcWalletPage.
* Remove usage in home layout.
* Remove usage in home route.
* Remove usage in tokens route.
* Remove usage in accounts layout.
* Remove usage in SnsWallet.
* Remove usage in SignInAccounts.

# Tests

* Remove describe blocks using the flag as false and also the block where the flag is set to true.

# Todos

- [ ] Add entry to changelog (if necessary).

Not yet necessary.

### CODE REVIEW

**Comment by lmuntaner** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #4407** <a href="https://app.graphite.dev/github/pr/dfinity/nns-dapp/4407?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/dfinity/nns-dapp/4407?utm_source=stack-comment).

**Comment by lmuntaner** [comment]
@dskloetd please review

File: frontend/src/tests/lib/pages/IcrcWallet.spec.ts [hunk 1]
```diff
@@ -321,18 +320,15 @@ describe("IcrcWallet", () => {
       );
     });
 
-    it("should navigate to accounts when account identifier is invalid", async () => {
+    it("should navigate to tokens when account identifier is invalid", async () => {
```
Comment (dskloetd) [inline frontend/src/tests/lib/pages/IcrcWallet.spec.ts:323]: Isn't this now a duplicate of the test below?

Comment (lmuntaner) [reply frontend/src/tests/lib/pages/IcrcWallet.spec.ts]: Yes, thanks! Removed.

**Pull request merged**